mne/transforms.py

```diff
@@ -77,8 +79,18 @@ class Transform(dict):
     ----------
     fro : str | int
         The starting coordinate frame.
+        Can be 'unknown', 'MEG device', 'isotrak', 'hpi', 'head',
+        'MRI (surface RAS)', 'MRI voxel', 'MRI slice', 'MRI display',
+        'CTF MEG device', 'CTF/4D/KIT head', 'RAS (non-zero origin)',
+        'MNI Talairach', 'Talairach (MNI z > 0)',
+        'Talairach (MNI z < 0)'
     to : str | int
         The ending coordinate frame.
+        Can be 'unknown', 'MEG device', 'isotrak', 'hpi', 'head',
+        'MRI (surface RAS)', 'MRI voxel', 'MRI slice', 'MRI display',
+        'CTF MEG device', 'CTF/4D/KIT head', 'RAS (non-zero origin)',
+        'MNI Talairach', 'Talairach (MNI z > 0)',
+        'Talairach (MNI z < 0)'
```

Comment:
It might be better to make a `Notes` section below and list them once down there

Reply:
+1 to add it in notes

something like "See notes for valid coordinate systems"

```
Notes
------
Valid coordinate systems are ...
```